tests/fixtures/locales.py

```diff
@@ -20,3 +21,16 @@ def locale1():
 def localeX():
     """Locale X - empty locale"""
     return Locale.objects.get(code="localeX")
+
+
+@pytest.fixture
+def locale_parts(projectX, localeX, locale1, entity0):
```

Comment:
And here as well. It's really unclear to me what this does, even after reading the code. Why is it called `parts`?

Reply:
same as above - it was named from the TestClass.setup that it was factored from